tests/TestCase.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Clue\Tests\React\Flux;
+
+use PHPUnit\Framework\TestCase as BaseTestCase;
+
+class TestCase extends BaseTestCase
+{
+    protected function expectCallableOnce()
+    {
+        $mock = $this->createCallableMock();
+
+        $mock
```

Comment:
nit: inconsistent indentation, see following methods.

Reply:
Everything should be fine now :+1: